src/components/discover-sheet/UniswapPoolsSection.js

```diff
@@ -253,7 +309,7 @@ export default function UniswapPools({
   }
 
   return (
-    <Column marginTop={32}>
+    <Column marginTop={25}>
```

Comment:
why so?

Reply:
I need to check. Actually this might be unnecessary 😅.